lib/experimental/button.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:html';
+
+import 'package:dart_pad/elements/elements.dart';
+import 'package:mdc_web/mdc_web.dart';
+
+/// Adds a ripple effect to material design buttons
+class MDCButton extends DButton {
```

Comment:
nit: Needs a copyright header in the file.

I love having a separate class for this.